### PR TITLE
Count_Inversion.cpp

### PR DESCRIPTION
**Pull request by mansikagrawal**
Fixes #2242 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

#### Files Added: Count_Inversion.cpp

Thanks!

### CODE REVIEW

**Comment by somya-kapoor** [comment]
Please write the issue number in Fixes, not the PR you previously created.

File: Count_Inversion/Count_Inversion.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+int inversions_count(int array[], int n) 
+{
+    int aux[n];
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main()
+{       
+    int n;
+    cin>>n;
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Please add space around the operator

File: Count_Inversion/Count_Inversion.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    for(int i=0;i<n;i++)
+    cin>>arr[i];
+    cout<<(inversions_count(arr, n))<<endl;
+    return 0;
+}
+
+/* Sample input 
+5
+1 20 6 4 5
+Sample Output
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Leave a line before output.

File: Count_Inversion/Count_Inversion.cpp [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main()
+{       
+    int n;
+    cin>>n;
+    int arr[n];
+    for(int i=0;i<n;i++)
+    cin>>arr[i];
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around the operator

File: Count_Inversion/Count_Inversion.cpp [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    cout<<(inversions_count(arr, n))<<endl;
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around the operator

File: Count_Inversion/Count_Inversion.cpp [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    int aux[n];
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main()
+{       
+    int n;
+    cin>>n;
+    int arr[n];
+    for(int i=0;i<n;i++)
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around the operator

**Comment by mansikagrawal** [comment]
@asha15  Please check and tell the other thing to be changed.

**Comment by somya-kapoor** [comment]
Please add space around the operator (<<.>>)

**Comment by mansikagrawal** [comment]
@somya-kapoor  Mam please check any further changes required.

File: Count_Inversion/Count_Inversion.cpp [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    int aux[n];
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main ()
+{       
+    int n;
+    cin >> n;
+    int arr[n];
+    for(int i=0; i<n; i++)
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around the operator

Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around the operator.

File: Count_Inversion/Count_Inversion.cpp [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    cin >> arr[i];
+    cout << (inversions_count(arr, n))<<endl;
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Space around <<

**Comment by somya-kapoor** [comment]
Good work!!
Please squash the commits after making changes.

**Comment by mansikagrawal** [comment]
@somya-kapoor  Mam actually new to this open source. How to do this ?

**Comment by somya-kapoor** [comment]
Please refer http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Pull request closed without merging**